<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FenAccueil</class>
 <widget class="QDialog" name="FenAccueil">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FenAccueil</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="adresseIPDuAgendailerLabel">
       <property name="text">
        <string>Adresse IP du Agendailer :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="ipAddrOfServer"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="portLabel">
       <property name="text">
        <string>Port :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="portOfServer">
       <property name="minimum">
        <number>1024</number>
       </property>
       <property name="maximum">
        <number>65536</number>
       </property>
       <property name="value">
        <number>50885</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="nomDUtilisateurLabel">
       <property name="text">
        <string>Nom d'utilisateur</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="UserName"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="motDePasseLabel">
       <property name="text">
        <string>Mot-de-passe</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="Password">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="connexion">
     <property name="text">
      <string>Connexion</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Debug</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPlainTextEdit" name="DebugConsole"/>
      </item>
      <item>
       <widget class="QPushButton" name="ping">
        <property name="text">
         <string>Ping</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="auth">
        <property name="text">
         <string>Authentification</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="OpenFenDevoir">
        <property name="text">
         <string>Ouvrir FenDevoir</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
